microsoft-edge/devtools-guide-chromium/experimental-features.md

```diff
@@ -0,0 +1,69 @@
+---
+description: The latest experimental features in Edge DevTools
+title: Experimental features
+author: hxlnt
+ms.author: raweil
+ms.date: 06/15/2020
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, experiment
+---
+
+# Experimental features
+
+The Edge DevTools periodically offers experimental features that are released while still in development. By enabling an experiment, you are able to test and [provide early feedback](#Providing-feedback-on-experimental-features) on these features before they are released broadly.
+
+Experiments are available in all channels of Edge. Use the Edge Canary channel to get the very latest experiments.
```

Comment:
Do we want to address Chromium experiments in any way? (Not sure if it's important, but customers may notice we're only documenting a subset of what shows up in the experiments settings tab)

Reply:
I would vote to:
1. Rename "Current experimental features" heading to "Selected experimental features" to denote that we're highlighting a subset.
2. At a later time, consider going back and filling in content on some of the older experiments. (There are 30+ experiments, and there might be considerable research to add them all in one go.)

Reply:
Thoughts?